frame/base-fee/src/lib.rs

```diff
@@ -303,6 +313,7 @@ mod tests {
 
 	frame_support::parameter_types! {
 		pub IsActive: bool = true;
+		pub DefaultBaseFeePerGas: U256 = U256::from(100_000_000_000 as u128);
```

Comment:
Shouldn't this be 1_000_000_000 ? (1Gwei)

Reply:
It's only the default one if there is no value in storage